api/dump.go

```diff
@@ -80,11 +75,12 @@ func (c *Client) prepareDump(ctx context.Context, pid int32, dir string, opts *r
 	}
 	opts.ShellJob = proto.Bool(isShellJob)
 
-	// processname + datetime
-	// strip out non posix-compliant characters from the processname
-	formattedProcessName := regexp.MustCompile("[^a-zA-Z0-9_.-]").ReplaceAllString(*pname, "_")
+	// jobID + UTC time (nanoseconds)
+	// strip out non posix-compliant characters from the jobID
+	formattedProcessName := regexp.MustCompile("[^a-zA-Z0-9_.-]").ReplaceAllString(c.jobID, "_")
```

Comment:
@lianakoleva you can get rid of the regex and formatting in these lines too, that was just to deal w/ long paths in the process name